core/src/main/java/hudson/Proc.java

```diff
@@ -431,7 +433,7 @@ private static String calcName(String[] cmd) {
      * @deprecated as of 1.399. Replaced by {@link Launcher.RemoteLauncher.ProcImpl}
      */
     @Deprecated
-    public static final class RemoteProc extends Proc {
+    public static final class RemoteProc extends Proc implements ProcWithJenkins23271Patch {
```

Comment:
Very old obsolete class, but we have one usage: https://github.com/jenkinsci/perforce-plugin/blob/4beaabe78faf62a8cfad340e933f1ff0bd2d07c1/src/main/java/hudson/plugins/perforce/HudsonP4RemoteExecutor.java#L113